org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/LayoutUtils.java

```diff
@@ -401,13 +401,14 @@ public static Mask showWindowWithMask(Window window, Component ownWindow, Mask m
 	public static ISupportMask findMaskParent (Component child, Class<?> parentClass){
 		Component parent = child;
 		ISupportMask trueParent = null;		
-		while ((parent = parent.getParent()) != null){
+		while (parent.getParent() != null){

```

Comment:
this change is different from the original. should be `while(parent != null)` and change the `Component parent = child` above to `Component parent = child.getParent()`

Reply:
I cannot change the `Component parent = child` to `Component parent = child.getParent()` because in our case the `child` is `ISupportMask` so we can't skip it in the while loop.

Reply:
ok, done